be/src/vec/common/columns_hashing.h

```diff
@@ -227,11 +227,11 @@ struct HashMethodSingleLowNullableColumn : public SingleColumnMethod {
     HashMethodSingleLowNullableColumn(const ColumnRawPtrs& key_columns_nullable,
                                       const Sizes& key_sizes, const HashMethodContextPtr& context)
             : Base(get_nested_column(key_columns_nullable[0]), key_sizes, context),
-              key_columns(key_columns_nullable) {}
+              key_column(reinterpret_cast<const ColumnNullable*>(key_columns_nullable[0])) {}
```

Comment:
Maybe it is better to use `assert_cast` here?

Reply:
OK

Reply:
OK